src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -185,12 +187,24 @@ def execute(self):
         try:
             self.protectedLFNs = set(self.wmstatsSvc.getProtectedLFNs())
             # self.logger.debug("protectedLFNs: %s", pformat(self.protectedLFNs))
-
             if not self.protectedLFNs:
-                msg = "Could not fetch the protectedLFNs list from WMStatServer. "
+                msg = "Could not fetch the protectedLFNs list from Production WMStatServer. "
                 msg += "Skipping the current run."
                 self.logger.error(msg)
                 return summary
+
+            self.protectedLFNsT0 = set()
+            if self.msConfig['enableT0WMStats']:
+                self.protectedLFNsT0 |= set(self.wmstatsSvcT0.getProtectedLFNsT0())
```

Comment:
I guess you don't want to have an OR here, right? Otherwise you will keep accumulating LFNs as long as the service is running.

Reply:
One can never jump over line 196 which is zeroing the protectedLFNsT0 set by default. Both operators `|=` and  `=` would work equally well here. I'll change it, no problem.